.github/workflows/cherrypick.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Cherry-pick Pull Request
+on:
+  pull_request_target:
+    types: [closed, labeled]
+
+permissions:
+  contents: read
+
+jobs:
+  cherrypick:
+    # NOTE: We currently need these permissions because we create pull request with the repo-scoped
+    # default token. We should in the future move to a PAT owned by lowrisc-bot and create pull request
+    # on its behalf.
+    permissions:
+      # Needed for the action to create branch.
+      contents: write
+      # Needed for the action to create a pull request.
+      pull-requests: write
+
+    name: Cherry-pick Pull Request
+    if: github.event.pull_request.merged == true && (github.event_name != 'labeled' || startsWith('CherryPick:', github.event.label.name))
```

Comment:
I'm not sure I understand this condition - won't it be `true` when any pull request is closed regardless of whether it has a `CherryPick:` label?

Reply:
Yes. The backport action will check the label to see whether backport is needed and what branch to backport to.

The `label` event is only present when the event is triggered by a label event, so it's not available when the triggering event is a merge.